Gruntfile.js

```diff
@@ -70,6 +70,12 @@ module.exports = function(grunt) {
                 return "url('" + imgUrl + "') repeat";
               }
               return 'initial';
+            },
+            assignFontSize: function(fontSize) {
+              if (fontSize) {
+                return fontSize + 'rem' || 'initial';
```

Comment:
Thank you so much, you've done very well 👍

Here are the missing things. I can finish it up but if you want to continue:
- All `px` units in LESS/CSS files should be replaced with `rem`s.
- `font_sizes.json` should contain the default sizes so that user will have an idea.

Btw, Hacktoberfest needs 4 pull-requests, not 4 merges, so you still have some chance 😄